app/helpers/application_helper.rb

```diff
@@ -274,9 +276,7 @@ def branch_info
   end
 
   def git_revision
-    ENV['DEPLOYMENT_ID']&.split('::').try(:[], 2)
-  rescue StandardError
-    'unknown revision'
+    Git.revision
```

Comment:
I could be wrong, but I don't think this will work once deployed.  The deployed app isn't an active git repo (I don't think), just a copy of the files.

Reply:
Oh, never mind, you aren't actually calling Git, just moved the method elsewhere.